src/main/java/net/unicon/lti/controller/lti/LTI3Controller.java

```diff
@@ -100,7 +100,12 @@ public String lti3(HttpServletRequest req, HttpServletResponse res, Model model)
         String link = req.getParameter("link");
         String altDomain = req.getHeader("x-formated-host");
```

Comment:
I know very little about this, but is this a typo? Formatted is spelled with 2 t's